content-src/components/DebugPage/DebugPage.js

```diff
@@ -4,6 +4,7 @@ const {selectNewTabSites, selectSpotlight} = require("selectors/selectors");
 const {SpotlightItem} = require("components/Spotlight/Spotlight");
```

Comment:
This is just a little debug thing to see all the types of highlights. When we get storybooks, this won't be necessary